versions.yaml

```diff
@@ -117,7 +117,7 @@ assets:
     architecture:
       aarch64:
         name: &default-image-name "ubuntu"
-        version: &default-image-version "jammy"
+        version: &default-image-version "oracular"
```

Comment:
Do you have any concerns with using a non-LTS version, or it is simply necessary until 26.04 to get multistrap working?

Reply:
I will report the issue to Ubuntu and see if we can get back to 24.04 at some of their coming releases.
Otherwise, we'll just need to bump on every release till 26.04 (and that's a commitment :-))